analysis/combiners.py

```diff
@@ -849,3 +802,37 @@ def metrics_names(self) -> List[str]:
 
     def explain_computation(self):
         pass
+
+
+def invert_error_quantiles(quantiles: List[float]) -> List[float]:
+    # The contribution bounding error is negative, so quantiles <0.5 for the
+    # error distribution (which is the sum of the noise and the contribution
+    # bounding error) should be used to come up with the worst error
+    # quantiles.
+    return [(1 - q) for q in quantiles]
+
+
+def compute_error_quantiles(quantiles: List[float], prob_to_keep: float,
```

Comment:
Nit: add a docstring, like
"""Computes quantiles of per partition errors for the sum of DP noise and contribution bounding error.""""